WalletWasabi.Fluent/Styles/HeaderedContentControl.axaml

```diff
@@ -0,0 +1,23 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  <Design.PreviewWith>
+    <Border Padding="20">
+      <HeaderedContentControl Header="Header" Classes="tile">
+        <TextBlock>Content</TextBlock>
+      </HeaderedContentControl>
+    </Border>
+  </Design.PreviewWith>
+
+  <Style Selector="HeaderedContentControl.tile">
```

Comment:
Why not make HeaderedContentControl part of TileControl, it's not used there and for other places we already have dedicated control ContentArea

Reply:
You're right. Better to include it as part of the control.